packages/keyda-server/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "keyda-server",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js",
-    "dev": "nodemon server.js"
+    "test": "jest --forceExit --coverage --verbose",
+    "start": "node src/server.js",
+    "dev": "nodemon src/server.js"
```

Comment:
Add `src/` prefix because of changing directory. and fixed test script statements.